build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -239,6 +239,7 @@ LIBRARY CoreGraphics
         _CGContextSetShadowProjectionTransform
         _CGContextCreateWithD2DRenderTarget
         _CGGetD2DFactory
+        _CGGetPixelFormatProperties
```

Comment:
Never mind, this is needed. since CGIWIC is in header implementation, if it get's imported in another framework. this needs to be exported. This should be solved when CGIWIC impl gets moved to a mm file (have some wip)